### ISSUE TITLE
Remove huge banner

### ISSUE THREAD

**Issue by mslinn** [chenjj/CORScanner#23]
Much as I like the `cors` Python module, and I appreciate the careful dedication of its author, the huge banner violates a core principal of UNIX / Linux program design. I wish the author would remove it, or only generate it when a user elects to display it by providing a command-line option. 

> Expect the output of every program to become the input to another, as yet unknown, program. 
> Don't clutter output with extraneous information.
>
>  – Doug McIlroy, the inventor of UNIX pipes and one of the founders of the UNIX tradition, from which Linux has evolved.

## Workaround
You can filter out the banner with the following code (might not work on macOS):

```shell
cors -u example.com  | sed -n '/^Starting/,$p'
```

I wrote about this [in my blog](https://www.mslinn.com/blog/2021/03/20/cors.html).

**Comment by chenjj** [comment]
Thanks for the suggestion, and thanks for sharing the interesting blog. When writing the tool, I didn't consider the scenario of redirecting output to another program, as it supports saving outputs to a JSON file (`cors -u example.com -o result.json`).   But what you said makes sense in some way. 

I have removed the banner in the Github repo. You can apply this change by using 

```
git clone https://github.com/chenjj/CORScanner
python cors_scan.py -u example.com
```

This change will be synced to the Pypi site soon (`pip install --upgrade cors`).

**Comment by chenjj** [comment]
`cors-0.9.10` is available now. Please check if it OK.  `pip install cors==0.9.10`

**Comment by mslinn** [comment]
Wow, you responded quickly!

```shell
$  pip install cors==0.9.10
Collecting cors==0.9.10
  Downloading cors-0.9.10-py3-none-any.whl (2.3 MB)
     |████████████████████████████████| 2.3 MB 13 kB/s
Requirement already satisfied: requests in /var/work/django/oscar/lib/python3.8/site-packages (from cors==0.9.10) (2.25.1)
...
```

I am not sure why `requirement already satisified` showed up. However, the proper version did install:

```shell
$ pip show cors
Name: cors
Version: 0.9.10
Summary: Fast CORS misconfiguration vulnerabilities scanner
Home-page: http://github.com/chenjj/CORScanner
Author: Jianjun Chen
Author-email: whucjj@hotmail.com
License: MIT
Location: /var/work/django/oscar/lib/python3.8/site-packages
Requires: future, argparse, gevent, tldextract, colorama, requests
Required-by:
```

The banner showed up again the next time I ran the program, so I uninstalled the PIP module and reinstalled. Problem solved.


## STDOUT and STDERR
This program's use of [STDOUT and STDERR](https://www.howtogeek.com/435903/what-are-stdin-stdout-and-stderr-on-linux/) is non-standard. Output is generated without regard to how the data might be consumed. For example, all of the output ifrom the following example is sent to STDOUT:

```shell
$ cors -u instagram.com
Starting CORS scan...
2021-03-21 08:28:00 WARNING Found misconfiguration! {"url": "https://instagram.com", "type": "special_characters_bypass", "credentials": "false", "origin": "https://instagram.com,.evil.com", "status_code": 200}
Finished CORS scanning...
```

I know this because piping STDOUT to `/dev/null` results in no output.
```shell
$ cors -u instagram.com > /dev/null
```

[`wget`](https://www.gnu.org/software/wget/manual/wget.html) is a similar program in that it reads online data, however it is a good example of how to write to STDOUT and STDERR.

## Git Tag
The latest git tag is v0.9.6. There is no corresponding tag for version 0.9.10. You might want to enhance your release script to create a git tag.


## Versioning
The `README.md` says:

> ## Version
> **Current version is 1.0**

This is clearly not true. Probably it is best to remove that section because it will be difficult to keep in perfect sync, and [point to the PyPi page](https://pypi.org/project/cors/). To achieve that, on the GitHub page for your project, click the gear icon next to the word About, then set **Website** to `https://pypi.org/project/cors/`

**Comment by mslinn** [comment]
I updated my blog post to reflect the changes you made, and reworked the material. Since you are an expert on CORS, I wonder if you would be so kind as to check what I wrote for accuracy. In particular, I am concerned that the section entitled [Content-Type Header](https://mslinn.com/blog/2021/03/20/cors.html#ctype) might not be 100% accurate.

**Comment by chenjj** [comment]
Thanks for your suggestions.

> STDOUT and STDERR

Just double-check, are you suggesting we should write `Starting CORS scan...` and `Finished CORS scanning...` to STDEER? That seems to make sense. I will change this later.

> Git Tag

I will add one.

> Versioning

I will remove it.

> I wonder if you would be so kind as to check what I wrote for accuracy. In particular, I am concerned that the section entitled Content-Type Header might not be 100% accurate.

I think the words are right. But I'm not sure if there is a strong connection between CORS and Content-Type "response" header, although the CORS's standard discusses Content-Type "request" header in the preflight request definition. 

In my testing experiences, many CORS configured sites do use a Content-Type header with various values, but some don't include Content-Type response header at all. 

Hope that answers your question.

**Comment by chenjj** [comment]
@mslinn  I made some changes and published `cors-0.9.11`. Please check if they are OK. Thanks again for your suggestions.

**Comment by mslinn** [comment]
All your changes seem OK. One last suggestion:

Normally, `-v` enables verbose output (but this causes no extra output with `cors` that I could see). `-vv` enables more verbose output (but again no change in output that I could see). `-vvv` is usually maximal verbosity. I suggest you change the implementation and the help messages `-vvv` to just use `-v`. Don't mention higher levels of verbosity unless they actually do something.

**Comment by chenjj** [comment]
Thank you. Will address that in the next version.

**Comment by chenjj** [comment]
> Will address that in the next version.

Fixed in v1.0.0. Please check if it OK. `pip install cors==1.0.0`
